### PR TITLE
Adding space embedding example

### PR DESCRIPTION
**Pull request by datavistics**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,6 +80,20 @@ A couple of alternatives:
 
 See an example [in this PR](https://github.com/huggingface/blog/pull/1180). This is to be used sparingly :)
 
+## Embed a 🤗 space
```
Comment (pcuenca) [inline README.md:83]:
```suggestion
## Embed a Space

```

File: README.md [hunk 2]
```diff
@@ -80,6 +80,20 @@ A couple of alternatives:
...
+Just add:
```
Comment (pcuenca) [inline README.md:84]:
```suggestion
Just add:

```

**Review by pcuenca — APPROVED** [review]
Thanks, LGTM!

File: README.md (hunk 1 above)
Comment (datavistics) [reply README.md:83]: 😭

**Pull request merged**